playgame.py

```diff
@@ -29,27 +35,26 @@ def main(argv):
                          action='store_true',
                          help='additionally log bot errors to stderr')
 
-    args = parser.parse_args()
-    game = HanabiGame(args)
-    game.playGame(args)
+    return parser.parse_args()
 
-def prepBots(botNames):
+def prep_bots(bot_names):
     res = []
-    for botName in botNames:
-        bot_class = locate(botName)
+    for bot_name in bot_names:
```

Comment:
What do you think about expressing this as a [`map`](http://book.pythontips.com/en/latest/map_filter.html#map)?

```python
def prep_bots(bot_names):
    return map(lambda: bot_name: locate(bot_name), bot_names)
```

Reply:
yup. done